inc/html.class.php

```diff
@@ -1429,6 +1429,10 @@ static function generateMenuSession($force = false) {
                   }
                }
             }
+            // Setup menu is always last
+            $setup = $menu['config'];
```

Comment:
Not sure this will work on any PHP version.
Using a uksort seem, to me,more appropriate:
```php
uksort(
   $menu,
   function ($key_a, $key_b) {
       if ('config' === $key_a) {
          return 1;
       }
       if ('config' === $key_b) {
          return -1;
       }
       return 0;
   }
);
```

Reply:
From [php.net/manual](https://www.php.net/manual/en/function.uksort.php): 
```
Note:
If two members compare as equal, their relative order in the sorted array is undefined.
```
**Zero cannot be returned here, or else, the order can be randomly changed.** (at least so say the manual, I'm not sure).

On the other hand, I found in [stackoverflow](https://stackoverflow.com/questions/2359652/how-do-i-move-an-array-element-with-a-known-key-to-the-end-of-an-array-in-php) the following discussion about "my" idea:
> -- This does no longer work in PHP 7.1, it seems.
> -- What exactly not working in PHP 7.1 ? it seems to work fine for me.
> -- When I tested it, the element was inserted *at the same position as it was originally* and not at the end, looked like some caching feature for me, but I could be wrong.

There is this way to do it that seems safe after some research:
```php
    $categories = array_keys($menu);
    $menu += array_splice($menu, array_search('config', $categories, true), 1);
```
`array_search` will return the position of key 'config', `array_splice` will remove the $menu['config'] from the array and also return it to `+=` that will insert it on the end.

It works in 9.3.3.

Reply:
OK, it is a bit tricky but it will work.
Just precede this by a clear comment that says it is to be sure that config is the last item in menu.